src/components/Checkbox/Checkbox.tsx

```diff
@@ -19,19 +18,15 @@ export interface CheckboxProps extends Omit<ComponentProps<'input'>, 'type' | 'r
   color?: keyof FlowbiteColors;
 }
 
-export const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
-  ({ className, color = 'default', theme: customTheme = {}, ...props }, ref) => {
-    const theme = mergeDeep(getTheme().checkbox, customTheme);
-
-    return (
-      <input
-        ref={ref}
-        type="checkbox"
-        className={twMerge(theme.root.base, theme.root.color[color], className)}
-        {...props}
-      />
-    );
-  },
-);
-
-Checkbox.displayName = 'Checkbox';
+export const Checkbox: Component<CheckboxProps> = (props) => {
+  const mergedProps = mergeProps({ color: 'default', theme: {} }, props);
+  const { class: className, color, theme: customTheme, ...restProps } = mergedProps;
```

Comment:
You need to use `splitProps` instead of destructuring, for examples:

```
 const [local, props] = splitProps(merged, ["theme"]);
 const theme = mergeDeep(getTheme().card, local.theme);
 
    <input
      type="checkbox"
      class={twMerge(theme.root.base, theme.root.color[color], className)}
      {...props}
    /> 
```